packages/gitmoji-changelog-core/src/index.js

```diff
@@ -31,10 +32,17 @@ function makeGroups(commits) {
 }
 
 function sanitizeVersion(version) {
-  return semver.valid(version, {
-    loose: false,
-    includePrerelease: true,
-  })
+  try {
+    return semver.valid(version, {
+      loose: false,
+      includePrerelease: true,
+    })
+  } catch (e) {
+    if (e.name === 'TypeError') {
+      throw FunctionalError(e.message)
+    }
+    return null
```

Comment:
You may should throw the error here instead of returning null ? 
`throw e`

Reply:
Sure !